app/controllers/user/unlocks_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class User::UnlocksController < Devise::UnlocksController
+  # GET /resource/unlock/new
+  # def new
+  #   super
+  # end
+
+  # POST /resource/unlock
+  # def create
+  #   super
+  # end
+
+  # GET /resource/unlock?unlock_token=abcdef
+  # def show
+  #   super
+  # end
+
+  # protected
+
```

Comment:
Why all devise controllers are commented out?

Reply:
I accidentally generated 2 sets of devise controllers: user and users.

The valid one is 'users', and i've deleted the duplicate